consumer/consumer.go

```diff
@@ -41,16 +56,18 @@ type Consumer interface {
 // Example:
 //
 // kafkaConsumer, err := consumer.New(brokerCfg, storage)
-// if err != nil {
-//     panic(err)
-// }
+//
+//	if err != nil {
```

Comment:
is this indent intended?

Reply:
yes, LSP did it and I trust the engine :)